### DISCUSSION TITLE
Versioning question

### DISCUSSION THREAD

**Discussion by blackandred** [crazy-max/docker-pure-ftpd#60] [Q&A] [answered]
Hi,

Thanks for the image. It's a only one image for ARM I found. I'm currently using SFTP, but I need to migrate to FTP quickly, so I'm interested in this image.

By the way I noticed you have RC versions tagged. Is this intentional that RC versions are replacing a stable tag? :wink: 

eg. `1.0.49-RC14` -> `docker.io/crazymax/pure-ftpd:1.0.49`

In my opinion RC is like a testing version, and the fixed version tags like 1.0.49 should be immutable for stability.

**Comment by crazy-max** [comment, marked as answer]
@blackandred,

> By the way I noticed you have RC versions tagged. Is this intentional that RC versions are replacing a stable tag?

Yes it's intentional. `RC` tags are just there to mark changes made on the Docker image and have nothing to do with Pure-FTPd versioning. Take a look at the [workflow](https://github.com/crazy-max/docker-pure-ftpd/blob/1be738d68f1f56723580381fb7414bfdbb7fdbfb/.github/workflows/build.yml#L26-L48) for more info.
